language_names/manual.py

```diff
@@ -0,0 +1,33 @@
+import sqlite3
+import textwrap
+import csv
+
+
+def insert_values(c, filename):
+    c.execute(textwrap.dedent("""
+        CREATE TABLE IF NOT EXISTS {} (
+            id INTEGER PRIMARY KEY,
+            lg TEXT,
+            inLg TEXT,
+            name TEXT,
+            UNIQUE(lg, inLg) ON CONFLICT REPLACE);
+        """.format(filename)))
+    with open('{}.tsv'.format(filename), 'r') as f:
+        reader = csv.DictReader(f, delimiter='\t')
+        for row in reader:
+            c.execute('INSERT INTO {} VALUES (?, ?, ?, ?)'.format(filename), (None, row['lg'], row['inLg'], row['name']))
+
+
+def populate_database():
+    conn = sqlite3.connect('langNames.db')
+    c = conn.cursor()
+    c.execute("""PRAGMA foreign_keys=OFF;""")
```

Comment:
This line and below and can use normal strings.

Reply:
Yeah you are right. I will use normal strings